Assets/Scripts/UI/Tools/Map/AreaInfo.cs

```diff
@@ -16,24 +15,35 @@ public class AreaInfo : MonoBehaviour {
 	public Transform Pivot;
 	public ToggleGroup ToggleGrp;
 
-
 	List<AreaListObject> Created = new List<AreaListObject>();
 
+	
 	public static void CleanSelection()
 	{
 		HideArea = false;
 		SelectedArea = null;
 
+		if (Current != null)
+		{
+			Current.DeselectAll();
+		}
 	}
+	
 
 	private void Awake()
 	{
 		Current = this;
+		
+		// These controls are currently not being used. I am disabling them like this instead of in the Editor to make 
+		// it clear in the code that we aren't using them. If we continue to not use them, we should remove them entirely.
```

Comment:
I'm not a fan of leaving things in an intermediary state if there are not good reasons for it.
Experience shows that these things will never be addressed again. We should remove them entirely and when there is new functionality that needs a new button it will not be hard to add one again.

Reply:
Hidden elements removed, code adjusted